### PR TITLE
Update of the examples based on the reviews of the PyAlbany paper

### PR DESCRIPTION
**Pull request by kliegeois**
This PR updates the PyAlbany examples to follow reviewers's comments.

### CODE REVIEW

File: PyAlbany/examples/paper/.gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 steady2d*
 *.jpeg
 *.sh
-*.txt
\ No newline at end of file
+*.txt
+input_scalar_updated.yaml
```
Comment (mperego) [inline PyAlbany/examples/paper/.gitignore:5]: why did you add `input_scalar_updated.yaml` to `.gitignore`?

**Review by mperego — APPROVED** [review]
I left a comment. Other than that it looks good to me

File: PyAlbany/examples/paper/.gitignore (hunk 1 above)
Comment (kliegeois) [reply PyAlbany/examples/paper/.gitignore:5]: Because I used an input file `PyAlbany/examples/paper/input_scalar_to_update.yaml` as a "template" to generate an updated one (updated in the script) and then write it to the disc to use as input argument of the executable. We don't want to commit the updated one as it changes all the time.

This is in the context of the MC simulation using the C++ executable.

Comment (mperego) [reply PyAlbany/examples/paper/.gitignore:5]: I see. It makes sense.

**Comment by kliegeois** [comment]
Can I merge it @mperego ?

**Comment by mperego** [comment]
Sure!
